packages/wdio-cli/src/launcher.js

```diff
@@ -23,7 +23,7 @@ class Launcher {
         }
 
         this.interface = new CLInterface(config, specs)
-        config.runnerEnv.FORCE_COLOR = this.interface.hasAnsiSupport
+        config.runnerEnv.FORCE_COLOR = Number(this.interface.hasAnsiSupport)
```

Comment:
`this.interface.hasAnsiSupport` is a boolean, why do we cast it to integer?

Reply:
The supports color package specifically looks for a 0 and not false on FORCE_COLOR.

https://github.com/chalk/supports-color/blob/master/index.js#L19